### ISSUE TITLE
I3OpNewServiceFactory is dead code (Trac #2131)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2123]
This class appears to have been entirely superseded by `I3ServiceFactory`. I cannot find any users in combo or even jeb, so I'm not sure a deprecation period is necessary. Can we just remove this?

Migrated from https://code.icecube.wisc.edu/ticket/2131
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:43", 
    "description": "This class appears to have been entirely superseded by `I3ServiceFactory`. I cannot find any users in combo or even jeb, so I'm not sure a deprecation period is necessary. Can we just remove this?", 
    "reporter": "cweaver", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067283824229", 
    "component": "combo core", 
    "summary": "I3OpNewServiceFactory is dead code", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2018-01-18T23:07:28", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180523T18:50:27: 
* olivas changed resolution from "" to "fixed"
* olivas changed status from "new" to "closed"
* olivas changed owner from "" to "olivas"
* olivas commented:

> In changeset 3063:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="3063"
> removing dead code.  closes #2123.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:04:21: nwhitehorn commented:

> In changeset 168857:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168857"
> removing dead code.  closes #2123.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:43: 
* olivas changed milestone from "IceRec 5.3" to ""
* olivas commented:

> Milestone IceRec 5.3 deleted
